src/windows/CameraProxy.js

```diff
@@ -710,7 +710,10 @@ function takePictureFromCameraWindows(successCallback, errorCallback, args) {
     var UIMaxRes = WMCapture.CameraCaptureUIMaxPhotoResolution;
     var totalPixels = targetWidth * targetHeight;
 
-    if (totalPixels <= 320 * 240) {
+    if (targetWidth == -1 && targetHeight == -1) {
```

Comment:
If targetWidth and targetHeight are not specified, they are initialized to -1. So even if we initialize, it will be changed to verySmallQvga in the present if condition.